packages/block-library/src/separator/edit.js

```diff
@@ -58,18 +58,32 @@ export default function SeparatorEdit( {
 
 	return (
 		<>
-			<InspectorControls group="advanced">
-				<HTMLElementControl
-					tagName={ tagName }
-					onChange={ ( value ) =>
-						setAttributes( { tagName: value } )
-					}
-					clientId={ clientId }
-					options={ [
-						{ label: __( 'Default (<hr>)' ), value: 'hr' },
-						{ label: '<div>', value: 'div' },
-					] }
-				/>
+			<InspectorControls>
+				<PanelBody title="Settings">
+					<SelectControl
+						label="HTML Element"
+						value={ tagName }
+						onChange={ ( value ) =>
+							setAttributes( { tagName: value } )
+						}
+						clientId={ clientId }
```

Comment:
The `clientId` is an `HTMLElementControl` component-specific prop; it's not required here.

Reply:
Change has been pushed. Thanks @Mamaduka.